app/scripts/modules/appengine/pipeline/stages/editLoadBalancer/appengineEditLoadBalancerStage.ts

```diff
@@ -30,7 +28,8 @@ class AppengineEditLoadBalancerStageCtrl {
     });
   }
 
-  public editLoadBalancer(loadBalancer: ILoadBalancer, index: number) {
+  // TODO: ANG, come back to this, remove parameter?
+  public editLoadBalancer(_loadBalancer: ILoadBalancer, index: number) {
```

Comment:
`_loadBalancer` isn't needed here, but there would need to be a corresponding change in the template from which this method is called.